Policies/github-corporate-terms-of-service.md

```diff
@@ -332,16 +348,16 @@ Subject to the limitations above, you agree to indemnify us and us hold harmless
+We’ll defend or settle, at our option and expense, any third-party claim brought against you to the extent that it’s based on an allegation that GitHub’s service infringes a copyright or misappropriates a trade secret of any third party (each, a “Claim”), and, subject to Section N, above, Limitation of Liability, we’ll pay all damages and costs (including reasonable legal fees) finally awarded by a court of final appeal attributable to such a Claim, provided that you notify us in writing of any such Claim as soon as reasonably practicable and allow us to control, and reasonably cooperate with us in the defense of, any such Claim and related settlement negotiations.
 
-You understand that we’ll have no obligation to indemnify you for any claim (1) if the total aggregate fees received by GitHub with respect to your subscription to the services in the 12 month period immediately preceding the claim is less than US $50,000; (2) if the services are modified by any party other than GitHub, but solely to the extent the alleged infringement is caused by such modification; (3) if the services are used in combination with any services, software, or equipment without prior written authorization by GitHub, but solely to the extent the alleged infringement is caused by such combination; (4) to unauthorized use of the services; (5) to any Claim arising as a result of (y) your Content (or circumstances covered by your indemnification obligations in this Section Q (Release and Indemnification: Customer’s Indemnification)) or (z) any third-party deliverables or components contained with the services; or (6) if you settle or make any admissions with respect to a claim without GitHub’s prior written consent.
+You understand that we’ll have no obligation to indemnify you for any claim (1) if the total aggregate fees received by GitHub with respect to your subscription to the services in the 12 month period immediately preceding the claim is less than US $50,000; (2) if the services are modified by any party other than GitHub, but solely to the extent the alleged infringement is caused by such modification; (3) if the services are used in combination with any services, software, or equipment without prior written authorization by GitHub, but solely to the extent the alleged infringement is caused by such combination; (4) to unauthorized use of the services; (5) to any Claim arising as a result of (y) your Content (or circumstances covered by your indemnification obligations in this Section O (Release and Indemnification: Customer’s Indemnification)) or (z) any third-party deliverables or components contained with the services; or (6) if you settle or make any admissions with respect to a claim without GitHub’s prior written consent.
```

Comment:
Why are the two subsections of (5) “(y)” and ”(z)”? Wouldn’t it make more sense to label them as “(a)” and “(b)”? Also, Why not break this out into a multiline list for readability?

<details><summary>Example:</summary><blockquote>

You understand that we’ll have no obligation to indemnify you for any claim:
1. if the total aggregate fees received by GitHub with respect to your subscription to the services in the 12 month period immediately preceding the claim is less than US $50,000;
1. if the services are modified by any party other than GitHub, but solely to the extent the alleged infringement is caused by such modification;
1. if the services are used in combination with any services, software, or equipment without prior written authorization by GitHub, but solely to the extent the alleged infringement is caused by such combination;
1. to unauthorized use of the services;
1. to any Claim arising as a result of
   1. your Content (or circumstances covered by your indemnification obligations in this Section O (Release and Indemnification: Customer’s Indemnification)) or
   1. any third-party deliverables or components contained with the services; or
1. if you settle or make any admissions with respect to a claim without GitHub’s prior written consent.

</blockquote></details>

<details><summary>Source:</summary>

```markdown
You understand that we’ll have no obligation to indemnify you for any claim:
1. if the total aggregate fees received by GitHub with respect to your subscription to the services in the 12 month period immediately preceding the claim is less than US $50,000;
1. if the services are modified by any party other than GitHub, but solely to the extent the alleged infringement is caused by such modification;
1. if the services are used in combination with any services, software, or equipment without prior written authorization by GitHub, but solely to the extent the alleged infringement is caused by such combination;
1. to unauthorized use of the services;
1. to any Claim arising as a result of
   1. your Content (or circumstances covered by your indemnification obligations in this Section O (Release and Indemnification: Customer’s Indemnification)) or
   1. any third-party deliverables or components contained with the services; or
1. if you settle or make any admissions with respect to a claim without GitHub’s prior written consent.
```

</details>